### PR TITLE
CI: fix grammar typo when test is succesful

### PR DESCRIPTION
**Pull request by BKPepe**
Maintainer: @aparcar

While adding test.sh to one of my packages, which I maintain, I noticed in the output, that there is typo. It's nitpick, but hopefully you don't mind that.

### CODE REVIEW

**Comment by BKPepe** [comment]
Ok, because it failed, I noticed there were some warnings:

https://github.com/openwrt/packages/actions/runs/310968253

so here goes another commit.

File: .github/workflows/multi-arch-test-build.yml [hunk 1]
```diff
@@ -46,7 +46,7 @@ jobs:
           PACKAGES="${PACKAGES:-vim tmux bmon}"
 
           echo "Building $PACKAGES"
-          echo "::set-env name=PACKAGES::$PACKAGES"
+          echo "name=PACKAGES::$PACKAGES" >> $GITHUB_ENV
```
Comment (aparcar) [inline .github/workflows/multi-arch-test-build.yml:49]:
```suggestion
          echo "PACKAGES=$PACKAGES" >> $GITHUB_ENV
```

Based on the [reference](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable) this fix seems wrong.

**Review by aparcar — CHANGES_REQUESTED** [review]

File: .github/workflows/multi-arch-test-build.yml (hunk 1 above)
Comment (BKPepe) [reply .github/workflows/multi-arch-test-build.yml]: Yes, you are correct. I noticed this as well when CI failed.

**Comment by BKPepe** [comment]
Now, it is ok. The rest fails also in other PRs. There is something messed up with SDK.

**Comment by aparcar** [comment]
Thanks

**Pull request merged**